configure.ml

```diff
@@ -194,6 +194,12 @@ let which prog =
 let program_in_path prog =
   try let _ = which prog in true with Not_found -> false
 
+let build_date =
+    try
```

Comment:
Note that indentation doesn't seem right here; not a merge blocker IMO but it would be great if we could fix it.

Reply:
fixed indentation